lib/ruby_lsp/listeners/document_link.rb

```diff
@@ -30,7 +30,7 @@ def gem_paths
                 lookup[spec.name] = {}
                 lookup[spec.name][spec.version.to_s] = {}
 
-                Dir.glob("**/*.rb", base: "#{spec.full_gem_path}/").each do |path|
+                Dir.glob("**/*.rb", base: "#{spec.full_gem_path.delete_prefix("//?/")}/").each do |path|
```

Comment:
In isolation, the `//?/` may be confusing. We could either add a comment, or extract it to give it a name, e.g. `WINDOWS_LONG_PATH_PREFIX`.